### PR TITLE
Updated tooltips in App.js

### PR DESCRIPTION
**Pull request by tuankhoin**
Added definition tooltips. You can now hover the mouse at keywords for definitions.
For documentation: [https://material-ui.com/components/tooltips/#arrow-tooltips]()
Contact: tuankhoin@student.unimelb.edu.au

### CODE REVIEW

**Comment by freddiewanah** [comment]
Hey, I'd like to review your PR, but it seems that there're some conflicts that need to be fix. And since you're introducing a new lib to the site, can you send some screenshot of the tooltip you've build and make sure that it doesn't influence our current component?

**Comment by tuankhoin** [comment]
Yes, I'll just screen cap and highlight what I've added to the file for you. I've noticed that there was some changes that I didn't make, probably because I was modifying on an older version of the repo.
The libraries that I've added, 4 in total:
image
Added HtmlToolTip:
image
Modified the stat function's return by adding tooltip inside each tag:
image
A demo of what I implemented:

**Comment by tuankhoin** [comment]
Hi, I've checked the conflict. Seems like it is not related to my modification, so I guess that it may have occurred because someone modified the origin while I was adding the code in. I have screenshot my modifications above for you. If you need anything, feel free to contact me. Have a good night Han!

**Comment by gamma032steam** [comment]
Resolved the conflict.

File: src/App.js [hunk 1]
```diff
@@ -387,14 +619,32 @@ function Stat({
           fColor={"#a93226"}
           increased={deadCountIncrease}
         >
-          Deaths
+        <HtmlTooltip
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Deaths</Typography>
+              <em>{"All cases that resulted in death, mostly found in the elderly."}</em>
```
Comment (gamma032steam) [inline src/App.js]: This one is probably unnecessary.

File: src/App.js [hunk 2]
```diff
@@ -387,14 +619,32 @@ function Stat({
...
           number={curedCount}
           fColor={"#00b321"}
           increased={curedCountIncrease}
         >
-          Recovered
+        <HtmlTooltip
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Recovered</Typography>
+              <em>{"All cases that are no longer infected."}</em>
```
Comment (gamma032steam) [inline src/App.js]: Good place to elaborate on what has to happen for a patient to be deemed 'recovered'.

File: src/App.js (hunk 1 above)
Comment (tuankhoin) [reply src/App.js]: It's just there to make sure that any tag will have a definition. I find it awkward to have only one with a definition when hovered. But you guys can remove it if you want to. Just a personal pov.

File: src/App.js (hunk 2 above)
Comment (tuankhoin) [reply src/App.js]: Tbh I'm not actually sure about its definition as well haha. What do you think it could be?

**Comment by tuankhoin** [comment]
Guys. Since I've unknowingly commiting to a new PR, and that the previous code that I was working on is an old version, I will now move to a new PR. You can head to PR #98 for further tooltips development.

**Pull request closed without merging**